test/team_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/team'
+require 'pry'
+
+class TeamTest < Minitest::Test
+
+  def setup
+    @team_data_1 = {
+                            :team_id=>"1",
+                            :franchise_id=>"23",
+                            :short_name=>"New Jersey",
+                            :team_name=>"Devils",
+                            :abbreviation=>"NJD",
+                            :link=>"/api/v1/teams/1"
+                            }
+
+    @team_data_2 = {
+                            :team_id=>"4",
+                            :franchise_id=>"16",
+                            :short_name=>"Philadelphia",
+                            :team_name=>"Flyers",
+                            :abbreviation=>"PHI",
+                            :link=>"/api/v1/teams/4"
+                            }
+
+    @team_data_3 = {
+                            :team_id=>"26",
+                            :franchise_id=>"14",
+                            :short_name=>"Los Angeles",
+                            :team_name=>"Kings",
+                            :abbreviation=>"LAK",
+                            :link=>"/api/v1/teams/26"
+                            }
+
+    @team_1 = Team.new(@team_data_1)
+    @team_2 = Team.new(@team_data_2)
+    @team_3 = Team.new(@team_data_3)
```

Comment:
These look nice. Will want to add a couple more for the league stats stuff I have been writing